wordpress-mobile/WordPress-Android
# Create followers block

Fixes #8441 

Creates a Followers block with two tabs - WPCOM and Email followers, in Stats. Clicking the `View more` button redirects the user to the old Followers screen. 

I'm adding two new BlockItems - one to contains a centered piece of information and the second to contain labels for two columns items. We could consider renaming them. At the moment we already have more items than I expected and it might be hard to understand what they represent. 

To test:
* Open Stats
* You see a Followers block
* You can switch between WordPress.com and Email tabs
* Clicking ViewMore redirects you to the Followers detail screen
